client/components/main/Desktop/RequestTypeSelector/SelectedTypes.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
```

Comment:
This file can be a reusable component for displaying chips. We have a lot of use cases for displaying chips. It would improve our productivity and save time.

Reply:
Yeah definitely, agree 100%. Will do that.